src/views/ApartmentApp/index.js

```diff
@@ -133,11 +144,38 @@ export default class ApartApp extends Component {
     }
   }
 
+  /**
+   * Callback for changing the primary applicant
+   * @param {String} profile The StudentProfileInfo object for the person who is to be made the primary applicant
+   */
+  handleChangePrimary = profile => {
+    this.setState({ updating: true });
+    if (profile) {
+      if (this.state.applicants.includes(profile)) {
+        this.setState({ newPrimaryApplicant: profile, editDialogOpen: true });
+      }
+    }
+  };
+
+  handleChangePrimaryAccepted = () => {
+    if (this.state.newPrimaryApplicant) {
+      try {
+        this.saveApplication(this.state.newPrimaryApplicant.ID, this.state.applicants);
+      } catch (error) {
+        this.snackbarText = 'This feature is not yet implemented.';
```

Comment:
This is changed in a later branch, right? (One that was made when save is working?)

Reply:
Yes, I believe the branch for #911 uses the new `change-modifier` endpoint to change the primary applicant

Reply:
Ah yes, I see it. You moved it to components/StudentApplication/index.js. Looks good.